src/terraform/gke/gke.tf

```diff
@@ -5,15 +5,14 @@
 resource "google_container_cluster" "primary" {
   name     = var.cluster_name
   description = var.description
-  location = local.location
-  node_locations = local.node_locations
+  location = var.region
```

Comment:
Is our `zone` variable being used anywhere?

Reply:
In my original code I had `node_locations = [var.zone]`. Maybe we can bring it back so that the cluster gets created in a specific zone that the user specified.

Reply:
> Is our `zone` variable being used anywhere?

zone is not valid for gcp resource block anymore. We use node_locations instead, whose type is a list(string). This list contains the zone names.

Reply:
Let me know, if you would like me to enforce type directly in resource block as in [var.zone], or define a variable of type list(string) for zone, and assign to node_locations